luaui/Widgets/map_startbox.lua

```diff
@@ -2,7 +2,7 @@ function widget:GetInfo()
 	return {
 		name = "Start Boxes",
 		desc = "Displays Start Boxes and Start Points",
-		author = "trepan, jK",
+		author = "trepan, jK, Beherith",
 		date = "2007-2009",
 		license = "GNU GPL, v2 or later",
 		layer = 0,
```

Comment:
if you don't mind, add `depends = {"gl4"}` here so this won't make headless fail.

can use also `depends = {"gl"},` or `depends = {"shader"},` if you prefer a more lenient check.